### PR TITLE
feat: image push

### PR DESCRIPTION
**Pull request by 3Hren**
[asciicast](https://asciinema.org/a/VMs5PpDwdIdu9JSP07o31mEnH)

```bash
esafronov@esafronov-osx3:~|⇒  python -c "import random; print(random.choice(['antmat', 'noxiouz', 'sshaman1101', 'oopcode']))"
antmat
```

### CODE REVIEW

File: cmd/cli/commands/tasks.go [hunk 1]
```diff
@@ -4,15 +4,21 @@ import (
...
 	"fmt"
 	"io"
+	"os"
+	"strconv"
 	"strings"
 	"time"
 
 	ds "github.com/c2h5oh/datasize"
 	"github.com/docker/go-connections/nat"
+	"github.com/pkg/errors"
```
Comment (noxiouz) [inline cmd/cli/commands/tasks.go]: std errors?

File: cmd/cli/commands/tasks.go [hunk 2]
```diff
@@ -188,6 +194,111 @@ var taskStartCmd = &cobra.Command{
...
+		if len(args) != 2 {
+			return errNotEnoughArguments
+		}
+
+		dealId := args[0]
+		path := args[1]
+		file, err := os.Open(path)
+		if err != nil {
+			return err
+		}
```
Comment (noxiouz) [inline cmd/cli/commands/tasks.go]: defer file.Close()

**Comment by 3Hren** [comment]
Windows strikes back. Seems like I need to try out another progress bar library.

**Comment by 3Hren** [comment]
Fixed!

**Pull request merged**